data/registers/pwr_g0.yaml

```diff
@@ -33,20 +33,54 @@ block/PWR:
       byte_offset: 24
       access: Write
       fieldset: SCR
-    - name: PUCR
-      description: Power Port pull-up control register
-      array:
-        len: 6
-        stride: 8
+    - name: PUCRA
+      description: Power Port A pull-up control register
       byte_offset: 32
-      fieldset: PCR
-    - name: PDCR
-      description: Power Port pull-down control register
-      array:
-        len: 6
-        stride: 8
+      fieldset: PUCR
+    - name: PDCRA
```

Comment:
is there a way to make these an array but have the letters still show up?

Reply:
Not really possible, array indices are only integers. 

There's one hack which is to keep both the PUCR array reg and the unrolled PUCRx regs. I wouldn't do it though, unrolled registers bloat the PAC and make it compile slower. The convention we're using is to arrayify everything even if it means losing letters.